sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/MiscFunctionsSuite.scala

```diff
@@ -130,4 +130,49 @@ class MiscFunctionsSuite extends SparkFunSuite with ExpressionEvalHelper {
       }
     }
   }
+
+  test("aesEncrypt") {
+    checkEvaluation(Base64(AesEncrypt(Literal("ABC".getBytes),
+      Literal("1234567890123456".getBytes))), "y6Ss+zCYObpCbgfWfyNWTw==")
+    checkEvaluation(Base64(AesEncrypt(Literal("".getBytes),
+      Literal("1234567890123456".getBytes))), "BQGHoM3lqYcsurCRq3PlUw==")
+
+    // Before testing this, installing Java Cryptography Extension (JCE)
+    // Unlimited Strength Jurisdiction Policy Files first
+//    checkEvaluation(Base64(AesEncrypt(Literal("ABC".getBytes),
+//      Literal("12345678901234561234567890123456".getBytes))), "nYfCuJeRd5eD60yXDw7WEA==")
```

Comment:
I think it might be okay, as long as the failure is clear and it doesn't break other things when its not installed.  What happens when its not installed?